Notifications/notifications_actions_customization.md

```diff
@@ -12,99 +12,111 @@ This document is a starting point for engaging the community and standards bodie
 Native applications that also have a Web counterpart might want the behavior of latter to be as similar to the former as much as possible to provide a consistent user experience across various devices and platforms. VoIP (Voice over Internet Protocol) applications - e.g., Microsoft Teams, Google Meet, Zoom, etc - are no exception and have become increasingly popular and demanded for remote collaboration over the last couple of years.
 
 ![call_teams_notification](https://user-images.githubusercontent.com/80070607/180506585-73f05e29-5676-4686-88dc-bd2fa05b8e6d.png)  
+
 *Figure 1: Calling notification scenario.*
 
-We would like to propose an extension to the [Notifications API](https://notifications.spec.whatwg.org/) standard for incoming call scenarios to allow the notification action buttons to be customized and also allow the application to play a custom ringtone. This capability would make the incoming call notifications, which may require a faster immediate response, clearly distinguishable from the others to the user and would also contribute to increasing accessibility on the Web. Moreover, apps would still be able to use their trademark ringtones on the web and users would be capable of identifying the calling app just by listening to it.
+We would like to propose an extension to the [Notifications API](https://notifications.spec.whatwg.org/) standard for incoming call scenarios to allow the notification action buttons to be customized and also allow the application to play a ringtone. This capability would make the incoming call notifications, which may require a faster immediate response, clearly distinguishable from the others to the user and would also contribute to increasing accessibility on the Web.
 
 ## Goals
 
-Propose an extension to the Notifications API to allow more customization for VoIP web apps. 
+Propose an extension to the Notifications API to allow more customization for VoIP Progressive Web Apps (PWAs). 
 
 ## Non-Goals
 
 Allow general web contents to arbitrarily change action button colors and sound sources without restrictions from the underly platforms.
 
 ## Use Cases 
 
-- A video conferencing website can produce its call notification pop-up with a green "accept" button and red "decline" button.
-- Web apps will be capable to change the notification action button colors if they feel like this will improve it UX.
+- A video conferencing PWA can produce its call notification pop-up with a green "accept" button and red "decline" button.
+- Web apps will be capable to change the notification action button colors if they feel like this will improve its UX.
```

Comment:
Should we also say something about the ringtone and the potential priority boost here?  What are the complete list of reasons an installed PWA would want to use this api?

Reply:
Removed the last bullet-point that seemed out of place and added another one that presents the ringtone/priority use cases. Thanks!